packages/plugins/typescript/resolvers/tests/ts-resolvers.config.customDirectives.spec.ts

```diff
@@ -62,7 +62,6 @@ describe('customDirectives.sematicNonNull', () => {
         nonNullableListWithNonNullableItemLevel0?: Resolver<Array<ResolversTypes['String']>, ParentType, ContextType>;
         nonNullableListWithNonNullableItemLevel1?: Resolver<Array<ResolversTypes['String']>, ParentType, ContextType>;
         nonNullableListWithNonNullableItemBothLevels?: Resolver<Array<ResolversTypes['String']>, ParentType, ContextType>;
-        __isTypeOf?: IsTypeOfResolverFn<ParentType, ContextType>;
```

Comment:
This is a new test from `master`. 
`__isTypeOf` is not required here after this PR in the feature branch: https://github.com/dotansimha/graphql-code-generator/pull/10283